src/main/java/info/u_team/useful_railroads/UsefulRailroadsMod.java

```diff
@@ -10,6 +14,11 @@
 	
 	public UsefulRailroadsMod() {
 		JarSignVerifier.checkSigned(MODID);
+
+		ModLoadingContext.get().registerConfig(ModConfig.Type.COMMON, Config.COMMON_CONFIG);
+
+		Config.loadConfig(Config.COMMON_CONFIG, FMLPaths.CONFIGDIR.get().resolve("usefulrailroads-common.toml"));
```

Comment:
Im not sure what 
```java
Config.loadConfig(Config.COMMON_CONFIG, FMLPaths.CONFIGDIR.get().resolve("usefulrailroads-common.toml")); ```
should do. The config is loaded automatically if its registered